src/pages/supertokens.tsx

```diff
@@ -0,0 +1,14 @@
+import { NextPage } from "next";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+// redirects the generic url /supertokens to the default network specific page
+// if changing this redirect, also apply those changes in SuperTokensLegacyRedirect
+const SuperTokensRedirect: NextPage = () => {
+    const router = useRouter();
+    useEffect(() => void router.replace("/matic/supertokens") ,[]);
```

Comment:
Because this is duplicated then it _could_ be moved into a custom React hook in style of `useRedirectToSuperTokensPage` which encapsulates the following:
```ts
const router = useRouter();
useEffect(() => void router.replace("/matic/supertokens") ,[]);
```
but this is such a cheap duplication that I wouldn't bother. Rule of three, you know...